### DISCUSSION TITLE
What is the symbols.zip used for?

### DISCUSSION THREAD

**Discussion by migf1** [Open-Shell/Open-Shell-Menu#1598] [Q&A] [answered]
Hey, I just downloaded & installed the latest stable version of OpenShell on Win 8.1 Pro, everything looks fine so far (amazing utility, thank you very much).

I also downloaded the symbols.7z file, available in the release Assets but I'm not sure what I'm supposed to do with it. It is full of dll, exe, and pdb files, do I need any of that?

Thanks!

**Comment by ge0rdi** [comment, marked as answer]
> I also downloaded the symbols.7z file, available in the release Assets but I'm not sure what I'm supposed to do with it. It is full of dll, exe, and pdb files, do I need any of that?

It contains symbol files necessary for debugging.

You don't need it at all.

**Comment by migf1** [comment]
Thank you!
